routes/index.js

```diff
@@ -255,6 +255,43 @@ module.exports = function () {
 
   // ===== /end docs
 
+  // Awesome
+  router.get(['/awesome', '/awesome/*'], (req, res) => {
+    const BASE = 'https://raw.githubusercontent.com/datahubio/awesome/master/';
+    let path = urllib.parse(req.path).path;
```

Comment:
I don't think we need to parse path here. Let's just have a single path value (if there) parsed by routes.